pt_BR.ini

```diff
@@ -315,7 +317,8 @@ Remove From Recent = Remover dos recentes
 SaveData = Dados de gravação do jogo
 Show In Folder = Mostrar na pasta
 USA = USA
-Use UI background = Usar UI background
+Use UI background = Usar background
```

Comment:
The reason it has UI in English is to make sure you don't think it will change your Android Wallpaper.  It only affects PPSSPP.  But too wide and the button will look bad.

-[Unknown]